### PR TITLE
qocamlbrowser 0.2.10

### PR DESCRIPTION
**Pull request by Kakadu**

### CODE REVIEW

**Comment by mseri** [comment]
There are some issues with `qt` on `osx` (the PATH in that case has to be `/usr/local/opt/qt/bin`) and I think the `compiler-libs` in ocaml 4.07.0 are changed slightly as the compilation fails with
```
- File "src/tree.ml", line 16, characters 26-36:
- Error: Unbound record field Ident.name
```

**Comment by Kakadu** [comment]
@mseri How did you find out an issue with OSX? It seems that CI doesn't have a build for that..

**Comment by mseri** [comment]
Travis has an osx build: https://travis-ci.org/ocaml/opam-repository/jobs/405691303

File: packages/qocamlbrowser/qocamlbrowser.0.2.10/opam [hunk 1]
```diff
@@ -4,9 +4,10 @@ homepage: "https://github.com/Kakadu/QOcamlBrowser_quick/"
 bug-reports: "https://github.com/Kakadu/QOcamlBrowser_quick/issues"
 authors: "Kakadu"
 build: [
-  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"]
-  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]
+  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"]
```
Comment (mseri) [inline packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: There is a way to make these depend on the OS, so you could use the right PATH in darwin and linux separately. I'll try to dig an example

Comment (mseri) [reply packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: See https://github.com/ocaml/opam-repository/blob/master/packages/conf-blas/conf-blas.1/opam#L9 for example

File: packages/qocamlbrowser/qocamlbrowser.0.2.10/opam [hunk 2]
```diff
@@ -3,9 +3,11 @@ maintainer: "kakadu.hafanana@gmail.com"
...
 bug-reports: "https://github.com/Kakadu/QOcamlBrowser_quick/issues"
 authors: "Kakadu"
+dev-repo: "git://github.com/Kakadu/QOcamlBrowser_quick"
+
 build: [
-  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"]
-  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"] {os != "darwin" }
+  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:$PATH make"]                      {os  = "darwin" }
 ]
```
Comment (mseri) [inline packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: I think this should be:

```
["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"] {os != "darwin" }
["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]        {os != "darwin" }
["sh" "-exc" "PATH=/usr/local/opt/qt/bin:$PATH ./configure"]               {os  = "darwin" }
["sh" "-exc" "PATH=/usr/local/opt/qt/bin:$PATH make"]                      {os  = "darwin" }
```

**Comment by mseri** [comment]
Thanks. Looks good!

File: packages/qocamlbrowser/qocamlbrowser.0.2.10/opam [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [
+  [make "uninstall"]
+]
+depends: [
+  "lablqml" {>= "0.5.1"}
+  "conf-qt" {>= "5.2.1"}
+  "ocamlbuild" {build}
+]
+available: (ocaml-version >= "4.07.0")
```
Comment (kit-ty-kate) [inline packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: any reasons why this restriction? I can't see any recent commit using 4.07 features

Comment (mseri) [reply packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: Oops, I did not notice that 😱 
Maybe it was meant to be `<=`?

Comment (mseri) [reply packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: I'll send a PR

Comment (mseri) [reply packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: https://github.com/ocaml/opam-repository/pull/12384

Comment (Kakadu) [reply packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: By some reason from https://github.com/ocaml/opam-repository/pull/12367#issuecomment-405916478 I thought that `Ident.name` fix introduces `>= 4.07` constraint. It seems that I was wrong...

**Pull request merged**